services/src/api/handlers/layers.rs

```diff
@@ -187,6 +187,10 @@ async fn get_layer_providers<C: ApplicationContext>(
         })
         .await?
     {
+        if provider_listing.priority == i16::MIN {
```

Comment:
unrelated to this line: The Data Catalogue will always be on top, right? Is that ok?

Reply:
yes it is. I think it should be always on top since it is the main entry point.